control/content/js/aiStateSeeder.js

```diff
@@ -0,0 +1,149 @@
+const initAiStateSeeder = {
+    aiStateSeeder: null,
+    dbProvider: 'datastore',
+    tagCollectionName: null,
+    generateOptions: {
+        userMessage: 'Generate a dummy list of contacts, up to 5 contacts',
```

Comment:
dummy with sample
no need for up to 5 contacts
for people working in <business-type>